File: rest-service/src/main/java/mb/lib/query/db/Column.java
package mb.lib.query.db;

interface Column
{
  interface MultiBlastJobs
  {
    String JobID     = "job_digest";
    String JobConfig = "job_config";
    String Query     = "query";
    String QueueID   = "queue_id";
    String ProjectID = "project_id";
    String Status    = "status";
    String CreatedOn = "created_on";
    String DeleteOn  = "delete_on";
  }

  interface MultiBlastUsers
  {
    String JobID           = "job_digest";
    String UserId          = "user_id";
    String Description     = "description";
    String MaxDownloadSize = "max_download_size";
    String RunDirectly     = "run_directly";
  }

  interface MultiBlastJobToJobs
  {
    String JobID        = "job_digest";
    String ParentDigest = "parent_digest";
    String Position     = "position";
  }

  interface MultiBlastJobToTargets
  {
    String JobID      = "job_digest";
    String Organism   = "organism";
    String TargetFile = "target_file";
  }
}

File: rest-service/src/main/java/mb/lib/query/db/Column.kt
package mb.lib.query.db

interface Column {
  interface MultiBlastJobs {
    companion object {
      const val JobID = "job_digest"
      const val JobConfig = "job_config"
      const val Query = "query"
      const val QueueID = "queue_id"
      const val ProjectID = "project_id"
      const val Status = "status"
      const val CreatedOn = "created_on"
      const val DeleteOn = "delete_on"
    }
  }

  interface MultiBlastUsers {
    companion object {
      const val JobID = "job_digest"
      const val UserId = "user_id"
      const val Description = "description"
      const val MaxDownloadSize = "max_download_size"
      const val RunDirectly = "run_directly"
    }
  }

  interface MultiBlastJobToJobs {
    companion object {
      const val JobID = "job_digest"
      const val ParentDigest = "parent_digest"
      const val Position = "position"
    }
  }

  interface MultiBlastJobToTargets {
    companion object {
      const val JobID = "job_digest"
      const val Organism = "organism"
      const val TargetFile = "target_file"
    }
  }
}
